$OpenBSD: patch-configure,v 1.4 2015/03/29 23:36:29 jeremy Exp $

We run configure with bundler exec, so we don't need bundler/setup.

Specify openbsd as the os so it doesn't use a version specific
value that would require bumping on OpenBSD version upgrades.

--- configure.orig	Tue Jun 10 20:27:20 2014
+++ configure	Sun Mar 29 09:11:23 2015
@@ -1,7 +1,7 @@
 #!/usr/bin/env ruby
 
 require 'rubygems'
-require 'bundler/setup'
+#require 'bundler/setup'
 require './rakelib/configure'
 require './rakelib/release'
 require './rakelib/build_signature'
@@ -212,7 +212,7 @@ class Configure
 
   def set_host
     /([^-]+)-([^-]+)-(.*)/ =~ @host
-    @cpu, @vendor, @os = $1, $2, $3
+    @cpu, @vendor, @os = $1, $2, 'openbsd'
 
     # TODO: For better cross-compiling support, it may be necessary to
     # use the feature facility to check for a define in the compiler.
